static/templates/subscription_invites_warning_modal.handlebars

```diff
@@ -0,0 +1,15 @@
+<div id='invites-warning-overlay' class='overlay new-style show' style='z-index:1000;'>
```

Comment:
Yeah that modal was also added by me, while adding this modal I noticed that the stream creation form in the back was also editable which should not be the case. Actually the problem is that bootstrap doesn't allow opening another modal while one is already visible. This was the quickest workaround I can think off for now, do you have some other trick to fix this? Regarding `subscription_stream_privacy_modal`, adding an overlay to it is on my TODO and will do that in a separate PR. :)